script_runner/tasks.py

```diff
@@ -284,6 +262,14 @@ def execute(script_path, ctx, process):
         proxy.close()
     except Exception:
         ctx.logger.warning('Failed closing context proxy', exc_info=True)
+    else:
+        ctx.logger.debug("Context proxy closed")
+
+    for t, name in [(stdout_consumer, 'stdout'), (stderr_consumer, 'stderr')]:
+        if t:
+            ctx.logger.debug('Joining consumer thread for %s', name)
+            t.join()
+            ctx.logger.debug('Consumer thread for %s ended', name)
```

Comment:
For readability, can you change `t` to something like `consumer`?

Reply:
Yes, one sec.

Reply:
Done.